spark-connect/index.md

```diff
@@ -0,0 +1,151 @@
+
+<img src="{{site.baseurl}}/images/spark-connect2.png" style="width: 100%; max-width: 500px;">
+
+## Benefits of Spark Connect
+
+Let’s now turn our attention to the benefits of the Spark Connect architecture.
+
+**Spark Connect workloads are easier to maintain**
+
+With the Spark JVM architecture, the client and Spark Driver must run identical software versions.  They need the same Java, Scala, and other dependency versions.  Suppose you develop a Spark project on your local machine, package it as a JAR file, and deploy it in the cloud to run on a production dataset.  You need to build the JAR file on your local machine with the same dependencies used in the cloud.  If you compile the JAR file with Scala 2.13, you must also provision the cluster with a Spark JAR compiled with Scala 2.13.
+
+Suppose you are building your JAR with Scala 2.12, and your cloud provider releases a new runtime built with Scala 2.13.  For Spark JVM, you need to update your project locally, which may be challenging.  For example, when you update your project to Scala 2.13, you must also upgrade all the project dependencies (and transitive dependencies) to Scala 2.13.  If some of those JAR files don’t exist, you can’t upgrade.
+
+In contrast, Spark Connect decouples the client and the Spark Driver, so you can update the Spark Driver including server-side dependencies without updating the client.  This makes Spark projects much easier to maintain.
+
+**Spark Connect lets you build Spark Connect Clients in non-JVM languages**
+
+Spark Connect decouples the client and the Spark Driver so that you can write a Spark Connect Client in any language.  Here are some Spark Connect Clients that don’t depend on Java/Scala:
+
+* [Spark Connect Python](https://github.com/apache/spark/tree/master/python/pyspark/sql/connect)
+* [Spark Connect Go](https://github.com/apache/spark-connect-go)
+* [Spark Connect Rust](https://github.com/sjrusso8/spark-connect-rs)
+
+For example, the Apache Spark Connect Client for Golang, [spark-connect-go](https://github.com/apache/spark-connect-go), implements the Spark Connect protocol and does not rely on Java.  You can use this Spark Connect Client to develop Spark applications with Go without installing Java or Spark.
+
+Here’s how to execute a query with the Go programming language using spark-connect-go:
+
+```
+spark, _ := sql.SparkSession.Builder.Remote(remote).Build()
+df, _ := spark.Sql("select * from my_cool_table where age > 42")
+df.Show(100, false)
+```
+
+When `df.Show()` is invoked, spark-connect-go processes the query into an unresolved logical plan and sends it to the Spark Driver for execution.
+
+spark-connect-go is a magnificent example of how the decoupled nature of Spark Connect allows for a better end-user experience.
+
+Go isn’t the only language that will benefit from this architecture.  The Spark Community is also building [a Rust ](https://github.com/sjrusso8/spark-connect-rs)Spark Connect Client.
```

Comment:
Also, I'd recommend to move this line to the reference section to be clear that this is no Apache Spark repository.